src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -2420,11 +2588,11 @@ class MavlinkStreamActuatorControlTarget : public MavlinkStream
 	uint64_t _att_ctrl_time;
 
 	/* do not allow top copying this class */
-	MavlinkStreamActuatorControlTarget(MavlinkStreamActuatorControlTarget &) = delete;
-	MavlinkStreamActuatorControlTarget &operator = (const MavlinkStreamActuatorControlTarget &) = delete;
+	MavlinkStreamActuatorControlTarget<N>(MavlinkStreamActuatorControlTarget<N> &) = delete;
```

Comment:
Why is this change templated?

Reply:
Agreed, apologies.  Fixed and pushed up.  Thanks for your review!